src/modules/internet/index.ts

```diff
@@ -109,6 +109,13 @@ export class InternetModule {
       );
     }
 
+    //local parts may not contain two or more consecutive . characters
+    localPart = localPart.replace(/\.{2,}/g, '.');
+
+    //local parts may not start with or end with a . character
+    localPart = localPart.replace(/^\./, '');
+    localPart = localPart.replace(/\.$/, '');
```

Comment:
This should also catch multiple dots:

```suggestion
    localPart = localPart.replace(/^\.+/, '');
    localPart = localPart.replace(/\.+$/, '');
```

Reply:
Alternatively:

```suggestion
    localPart = localPart.replace(/^\.+|\.+$/g, '');
```

Reply:
but if we already collapsed multiple dots into one on line 113, there can only be max 1 dot at start or end?

Reply:
Yeah, you are right.